_local.tf

```diff
@@ -6,22 +6,21 @@ resource "random_string" "s3_viewer" {
 
 locals {
   registry_url      = var.disable_ecr == false ? "${data.aws_caller_identity.current.account_id}.dkr.ecr.${var.region}.amazonaws.com" : "ghcr.io/cdcgov/phdi"
-  registry_auth     = data.aws_ecr_authorization_token.this.proxy_endpoint
   registry_username = data.aws_ecr_authorization_token.this.user_name
   registry_password = data.aws_ecr_authorization_token.this.password
-  service_data = {
+  service_data = var.service_data == {} ? {
     ecr-viewer = {
       short_name     = "ecrv",
       fargate_cpu    = 1024,
       fargate_memory = 2048,
-      min_capacity   = 1
-      max_capacity   = 5
+      min_capacity   = 1,
```

Comment:
Oh, no. This is going to be one of these stylistic areas that gives me fits. Is this JSON or YAML here? I've got a similar structure in `dibbs-azure` for the block definitions that uses no commas whatsoever. I'm in the camp that they clutter things unnecessarily unless they are explicitly needed by the language spec.